Stripe/STPOptimizationMetrics.h

```diff
@@ -0,0 +1,18 @@
+//
+//  STPOptimizationMetrics.h
+//  Stripe
+//
+//  Created by Ben Guo on 7/15/16.
+//  Copyright © 2016 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+extern NSString *const STPUserDefaultsKeyFirstAppOpenTime;
+extern NSString *const STPUserDefaultsKeyTotalAppOpenCount;
+extern NSString *const STPUserDefaultsKeyTotalAppUsageDuration;
+
+@interface STPOptimizationMetrics : NSObject
```

Comment:
I think it would be better to just use `STPOptimizationMetrics` as a singleton, and then you can migrate all of the notification logic here instead. It'll make the interfaces cleaner too.